tools/upgrade/steps/upgrade_50_replace_template

```diff
@@ -29,17 +29,8 @@ replace_template::run() {
 
     echo "      * Updating version to $tag"
     update_version $tag
-}
-
-extract_template_name() {
-    local file=$1
 
-    local ext="${file##*.}"
-    if [ $ext = "yml" ] || [ $ext = "yaml" ]; then
-        grep 'name:' $file | head -1 | sed -e 's/^[ ]*name:[ ]*//'
-    else
-        cat $file | jq -r .metadata.name
-    fi
+    exit 1
```

Comment:
this `exit 1` probably shouldn't be here

Reply:
oops. absolutely correct, left over from simulating a failure. 

Good find !